airflow-bi/Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN pip install \
       sqlalchemy \
       pyhive \
       gspread \
+      gspread_pandas \
```

Comment:
Can you introduce version here? or it will happen the same as with `pyarrow` when we have a major upgrade